src/views/RecIM/components/Forms/EditMatchStatsForm/index.js

```diff
@@ -0,0 +1,245 @@
+import { Grid } from '@mui/material';
+import { useState, useMemo, useEffect } from 'react';
+import GordonDialogBox from 'components/GordonDialogBox';
+import { InformationField } from '../components/InformationField';
+import { ConfirmationWindowHeader } from '../components/ConfirmationHeader';
+import { ConfirmationRow } from '../components/ConfirmationRow';
+import { ContentCard } from '../components/ContentCard';
+import GordonLoader from 'components/Loader';
+import { getMatchStatusTypes, updateMatchStats } from 'services/recim/match';
+
+const EditMatchStatsForm = ({
+  teamMatchHistory,
+  closeWithSnackbar,
+  openEditMatchStatsForm,
+  setOpenEditMatchStatsForm,
+}) => {
+  const [errorStatus, setErrorStatus] = useState({
+    Score: false,
+    StatusID: false,
+  });
+
+  const [loading, setLoading] = useState(true);
+  const [matchStatus, setMatchStatus] = useState([]);
+
+  useEffect(() => {
+    const loadData = async () => {
+      setLoading(true);
+      setMatchStatus(await getMatchStatusTypes());
+      setLoading(false);
+    };
+    loadData();
+  }, []);
+
+  const createMatchStatsField = [
+    {
+      label: 'Score',
+      name: 'Score',
+      type: 'text',
+      error: errorStatus.Name,
+      helperText: '*Required',
+    },
+    {
+      label: 'Sportsmanship',
+      name: 'Sportsmanship',
+      type: 'text',
+      error: errorStatus.Name,
+      helperText: '*Required',
+    },
+    {
+      label: 'Status',
+      name: 'StatusID',
+      type: 'select',
+      menuItems: matchStatus.map((type) => {
+        return type.Description;
+      }),
+      error: errorStatus.Name,
+      helperText: '*Required',
+    },
+  ];
+
+  const allFields = [createMatchStatsField].flat();
+
+  const currentInfo = useMemo(() => {
+    return {
+      TeamID: teamMatchHistory.TeamID,
+      Score: teamMatchHistory.TeamScore,
+      Sportsmanship: '',
```

Comment:
```suggestion
      Sportsmanship: 0,
```
despite sportsmanship not working fully at the moment, I don't know if TypeScript will be happy casting an empty string to a number. Once I implement the sportsmanship functionality, we might also get errors if we don't make the change now.